tools/util/include/cutlass/util/reference/device/tensor_relu.h

```diff
@@ -112,6 +112,62 @@ struct TensorReLuFunc {
   }
 };
 
+template <
```

Comment:
Could you move all the gelu related definitions in this file into a tensor_gelu.h file?